whirlpool/appliance.py

```diff
@@ -96,15 +101,16 @@ async def fetch_data(self):
             return False
 
         uri = f"{self._backend_selector.base_url}/api/v1/appliance/{self._said}"
-        async with async_timeout.timeout(30):
-            async with self._session.get(uri, headers=self._create_headers()) as r:
-                self._data_dict = json.loads(await r.text())
-                if r.status == 200:
-                    return True
-                elif r.status == 401:
-                    await self._auth.do_auth()
-
-                LOGGER.error(f"Fetching data failed ({r.status})")
+        for n in range(3):
```

Comment:
```suggestion
        try_count = 3
        for n in range(retry_count):
```
just to avoid magic numbers

Reply:
Should we make it a parameter (aka TRY_COUNT = 3 in the top of the file?) Probably should do the same to the write function... since I just copied it form there.